src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -322,7 +324,9 @@ private void ReplaceDisplay(Display newDisplay)
         /// <remarks>
         ///     The player must be in the <see cref="PlayerState.Idle"/> state.<br/>
         ///     The raw video feature(http://tizen.org/feature/multimedia.raw_video) is required if
-        ///     the display is created with <see cref="MediaView"/>.
+        ///     the display is created with <see cref="MediaView"/>.<br/>
+        ///     If user want to use video and UI sync mode, please use <see cref="Tizen.Multimedia.Display(NUI.Window, bool)"/>.(Since tizen 6.5)<br/>
+        ///     But in UI sync mode, please note that <see cref="Tizen.Multimedia.Player.DisplaySettings"/> is not supported.
```

Comment:
```suggestion
        ///     the display is created with <see cref="MediaView"/>.<br/>
        ///     If a user wants to use video and UI sync mode, use <see cref="Tizen.Multimedia.Display(NUI.Window, bool)"/>.(Since tizen 6.5)<br/>
        ///     But <see cref="Tizen.Multimedia.Player.DisplaySettings"/> is not supported in UI sync mode.
```

Reply:
Okay I'll fix it.